packages/xrpl/HISTORY.md

```diff
@@ -3,6 +3,10 @@
 Subscribe to [the **xrpl-announce** mailing list](https://groups.google.com/g/xrpl-announce) for release announcements. We recommend that xrpl.js (ripple-lib) users stay up-to-date with the latest stable release.
 
 ## Unreleased
+
+### NEW FEATURES
+* Add nfts_by_issuer data type
+
```

Comment:
this change is not needed, this file is the history for releases, not for individual changes. Good effort trying to document the change though 👍

Reply:
Each individual change edits this file so that it doesn't need to be done all at the end, though.